### ISSUE TITLE
multi instrument?

### ISSUE THREAD

**Issue by DefinitlyEvil** [calclavia/DeepJ#47]
can it possibly suppot multi-instruments in the future?

**Comment by calclavia** [comment]
@DefinitlyEvil Yes, definitely! It's a difficult research task though which will require some smart solution to get working.

**Comment by DefinitlyEvil** [comment]
@calclavia I am a undergraduate student majoring music composition and I love Bach's music. May I have your Skype or discord? I think I can give some suggestions to the model. ;)

**Comment by calclavia** [comment]
@DefinitlyEvil Discord Calclavia#9037
It's not as simple as just multiple networks, because each instrument needs to be conditioned on another instrument for harmony. It'll have to be done somewhat efficiently.

**Comment by DefinitlyEvil** [comment]
@calclavia then more inputs will be provided to the network, the instrumentation data. ;)
